rfcs/text/0000-planning-for-the-dcp.md

```diff
@@ -0,0 +1,211 @@
+### DCP PR:
+
+***Leave this blank until the RFC is approved** then the **Author(s)** must create a link between the assigned RFC number and this pull request in the format:*
+
+`[dcp-community/rfc#](https://github.com/HumanCellAtlas/dcp-community/pull/<PR#>)`
+
+# Planning for the DCP
+
+## Summary
+
+This RFC adopts and formalizes existing best project management practices and applications from the DCP community to define one process for planning and delivering product increments of the DCP *data as a service* on a regular cadence.
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED" "MAY", and "OPTIONAL" in this document are to be interpreted as described in BCP 14, [RFC2119](https://www.rfc-editor.org/rfc/rfc2119.txt), and [RFC8174](https://www.rfc-editor.org/rfc/rfc8174.txt) when, and only when, they appear in all capitals, as shown here.
+
+## Author(s)
+
+ [Trevor Heathorn](mailto:theathor@ucsc.edu)
+ 
+ [Brian Raymor](mailto:brianraymor@chanzuckerberg.com)
+
+## Shepherd
+
+[TBD](mailto:username@example.com)
+
+## Motivation
+
+***[Note - remove if approved]***
+*Early versions of this RFC were socialized with the DCP community during a PM call and at the DCP meeting in February 2019* 
+
+One of the **challenges** facing the DCP as it matures was identified in this [Update from HCA Science Governance](https://docs.google.com/presentation/d/13hZ7vmr371mN1YFjFyDN-6c0Ul95NwiJcVsjA3nJFVE/edit#slide=id.p3):
+
+> "A lack of reproducible planning process that prioritizes deliverables and ensures confidence that they will meet the needs of HCA"
+
+### DCP _data as a service_
+
+As a starting point, DCP should think *outside-the-box* and adopt one of the [principles of LeSS](https://less.works/less/principles/whole-product-focus.html) 
+>"Customers don’t buy a part of the product, but the whole product … there is no value in separate parts or half-working parts."
+
+Historically, ownership of the DCP software has been distributed across our canonical architectural diagram with its colorful *think-inside-the-boxes* (components). As a result, DCP planning and development have also been distributed to a high degree, resulting in questions or confusion about DCP priorities relative to component priorities with a subsequent lack of alignment for delivery of product increments, unless externally imposed by requests from HCA Science Governance. 
```

Comment:
Are you proposing to align development on practically all items (including those internal to teams) or only on cross-component issues?

Reply:
We're **not** proposing alignment on _practically all items (including those internal to teams)_. 

This is part of the reason that Sprint Planning is distributed rather than centralized. The proposal is that component teams prioritize their Sprint Backlog during their Sprint Planning based on DCP priorities reviewed during Refinement which may be cross-component epics, _significant features_, or DevSecOps requirements for a release.

The intent is to encourage conversations about priorities such as "My team cannot contribute to this DCP priority in the next Sprint _because ..._ or My team can only contribute 40% to this DCP priority in the next Sprint _because ..._